Contribute/how-to-write-workflows-major.md

```diff
@@ -91,15 +91,15 @@ Need to fix something you submitted?  No problem!  Just make your changes in the
 
 ## Making your next change
 
-Got more changes you need to make unrelated to this one? Switch back to the master branch, pull from the upstream repository to make sure that your fork is up to date, and check out a new branch.  Run the following commands in Git Bash:
+Got more changes you need to make unrelated to this one? Switch back to the default branch, pull from the upstream repository to make sure that your fork is up to date, and check out a new branch.  Run the following commands in Git Bash:
 
 ````
-git checkout master
-git pull upstream master
+git checkout main
+git pull upstream main
```

Comment:
@BillWagner do you know if most people using this external facing content working in repos that have already been renamed?

Reply:
That's a good question. How soon are the azure docs repos changing?

Would the right answer here be to show both? Happy to make that change.

Reply:
From what I've heard, there isn't a firm date, but they are working on it. Assuming this may take a while what do you think about adding a note along the lines of "If 'git checkout main' doesn't work, the default branch of the repository you are working in may not have been renamed. To verify this, try 'git checkout master"...?

Reply:
Great suggestion @Jim-Parker I made edits this afternoon.

I think this is ready now.